autonomi/tests/file.rs

```diff
@@ -2,18 +2,21 @@
 
 mod common;
 
-use crate::common::{evm_network_from_env, evm_wallet_from_env_or_default};
+#[cfg(feature = "files")]
 use autonomi::Client;
+#[cfg(feature = "files")]
 use std::time::Duration;
+#[cfg(feature = "files")]
+use test_utils::evm::get_funded_wallet;
+#[cfg(feature = "files")]
```

Comment:
Are the files feat flags necessary here as there is already a feat flag for files in place at the top of this file?